MagneticField/GeomBuilder/src/DD4hep_volumeHandle.cc

```diff
@@ -197,9 +203,483 @@ namespace {
 
 using namespace cms::dd;
 
-#include "buildBox.icc"
-#include "buildTrap.icc"
-#include "buildTubs.icc"
-#include "buildCons.icc"
+/*
```

Comment:
I think the code duplication below should be avoided. I don't understand why the `icc` files can't be used as they are.

Reply:
That was my question (see above) but Ianna pointed out to slight changes in interface, eg x() becoming halfX()
My suggestion is to extract the relevant parameters for each shape in *volumeHandle.cc  and pass them to the functions in the .icc files, as done already in buildPseudoTrap.icc. This way the actual implementation remains where it is and we just need to adjust the callers.